selftests/deployment/README

```diff
@@ -41,4 +41,4 @@ All-in-one execution in container:
 It may be useful to run a playbook in a fresh container.  One example
 of a onliner that can achieve that:
 
-  podman run --rm -ti fedora:30 /bin/bash -c 'dnf -y install git ansible && ansible-pull -v -U git://github.com/avocado-framework/avocado -i selftests/deployment/inventory -c local selftests/deployment/rpm-copr.yml'
+  podman run --rm -ti fedora:30 /bin/bash -c 'dnf -y install git ansible python3-pyyaml && ansible-pull -v -U git://github.com/avocado-framework/avocado -i selftests/deployment/inventory -c local selftests/deployment/rpm-copr.yml'
```

Comment:
I don't think that's needed... This is what I get from running the original command:

```
Installing       : python3-pyyaml-5.1-1.fc30.x86_64                     40/83 
```

Which makes sense because ansible is being installed from an RPM package, which should handle the depedencies.

Reply:
Yes, I agree. Actually, this was an old README. There is a new one on README.rst